pyfakefs/tests/fake_pathlib_test.py

```diff
@@ -467,35 +498,53 @@ def test_exists(self):
 
     def test_open(self):
         self.create_dir(self.make_path('foo'))
-        self.assertRaises(OSError,
+        self.assertRaises(IOError if TestCase.is_python2 else OSError,
```

Comment:
`IOError` is an alias for `OSError` in Python 3, so you may just check for `IOError`.